lib/esp32/espnow.toit

```diff
@@ -6,6 +6,87 @@ import io
 import monitor
 import monitor show ResourceState_
 
+/**
+ESP-NOW communication for ESP32 devices.
+
+ESP-NOW is a proprietary protocol developed by Espressif for
+  low-power, connectionless communication between ESP32 devices.
+
+Devices can either communicate by broadcasting messages to all
+  devices in range, or by sending messages to specific devices using
+  their MAC address. In both cases, a key can be used to encrypt the
+  messages.
+
+# Limitations
+
+Messages are at most 1470 bytes long. For broadcast messages there
+  is no guarantee that messages are correctly delivered. For direct
+  messages, the implementation ensures that the PHY layer has
+  received the message, but there is no guarantee that the message
+  is processed by the receiving device.
+
+ESP-NOW only supports 20 peers, although one of them can be the
```

Comment:
It feels weird to say that a peer can be an address.

Reply:
done.